### PR TITLE
Validate each item of `multiple` setting list

### LINKED ISSUE #52: Validator: check each item of `multiple` setting list
- [ ] Review Charts [docs](https://axibase.com/docs/charts/) and find particular use-cases, when check of each list item is useful.
- [ ] Split `multiple` setting into items and call [`checkType`](https://github.com/axibase/charts-language-service/blob/fc5ad37c7e6c252bb42f13e947f07dd1f53f4f83/src/setting.ts#L49) on each one.

### PR DESCRIPTION
**Pull request by PolinaShneider**
Closes #52

### CODE REVIEW

File: src/setting.ts [hunk 1]
```diff
@@ -68,7 +69,7 @@ export class Setting extends DefaultSetting {
         break;
       }
       case "number": {
-        const persent = /(\d*)%/.exec(this.value);
+        const persent = /(\d*)%/.exec(value);
```
[the remark this answers is not in the document]
Comment (PolinaShneider) [reply src/setting.ts]: If you do not mind, I want to introduce `percent` as data type via separate PR.

[the remark this answers is not in the document]
Comment (PolinaShneider) [reply src/setting.ts]: You are right, having for `percent` predefined `minValue = 0` and `maxValue = 1` would simplify the code.

File: src/validator.ts [hunk 2]
```diff
@@ -1125,7 +1125,41 @@ export class Validator {
...
+                    case "string":
+                        errorMessage = `${setting.displayName} can not contain empty elements`;
+                        break;
+                    case "number":
+                        errorMessage = `All elements of ${setting.displayName} must be real (floating-point) numbers`;
+                        break;
+                    case "integer":
+                        errorMessage = `All elements of ${setting.displayName} must be integers`;
+                        break;
+                    case "boolean":
```
Comment (annstriganova) [inline src/validator.ts]: We haven't settings, which are arrays of booleans.

File: src/validator.ts [hunk 3]
```diff
@@ -1125,7 +1125,41 @@ export class Validator {
...
+                        errorMessage = `All elements of ${setting.displayName} must be boolean values`;
+                        break;
+                    default:
+                        errorMessage = `Some of ${setting.displayName}'s elements have wrong type`;
```
Comment (annstriganova) [inline src/validator.ts]: Is there any case, when it's reachable?

File: src/setting.ts [hunk 4]
```diff
@@ -45,21 +45,22 @@ export class Setting extends DefaultSetting {
   /**
    * Checks the type of the setting and creates a corresponding diagnostic
    * @param range where the error should be displayed
+   * @param value setting value to check [optional]
    */
-  public checkType(range: Range): Diagnostic | undefined {
+  public checkType(range: Range, value: string = this.value): Diagnostic | undefined {
```
Comment (annstriganova) [inline src/setting.ts]: It seems that `range` is extra, please remove it from parameters, `this.textRange` is better to use instead.

Comment (PolinaShneider) [reply src/setting.ts]: Then it will be textRange of value, not setting. Is it okay for user perception?

**Pull request merged**